src/main/java/org/opensearch/jobscheduler/transport/response/ExtensionJobActionResponse.java

```diff
@@ -26,7 +28,25 @@
      * @throws IOException if serialization fails
      */
     public ExtensionJobActionResponse(T actionResponse) throws IOException {
-        super(JobDetailsService.convertParamsToBytes(actionResponse));
+        super(convertParamsToBytes(actionResponse));
+    }
+
+    /**
+     * Takes in an object of type T that extends {@link Writeable} and converts the writeable fields to a byte array
+     *
+     * @param <T> a class that extends writeable
+     * @param actionParams the action parameters to be serialized
+     * @throws IOException if serialization fails
+     * @return the byte array of the parameters
+     */
+    private static <T extends Writeable> byte[] convertParamsToBytes(T actionParams) throws IOException {
```

Comment:
Deja vu.  We definitely should find a central location for these util classes that both Extensions and Plugins can use.